Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -589,6 +589,60 @@ private IEnumerator TestTouchableDistances(BaseNearInteractionTouchable touchabl
 
             yield return PlayModeTestUtilities.HideHand(handedness, inputSim);
         }
+
+
+        /// <summary>
+        /// Test the SetTouchableCollider(BoxCollider collider) method by changing the size of the new
+        /// box collider and rotating the object. This test is checking if the NearInteractionTouchable plane
+        /// is the same size as the box collider and in front of the object.
+        /// </summary>
+        [UnityTest]
+        public IEnumerator NearInteractionTouchableSetTouchableCollider()
+        {
+            GameObject cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            cube.transform.position = new Vector3(0, 0, 1.5f);
+            cube.transform.localScale = new Vector3(0.3f, 0.3f, 0.3f);
+
+            cube.AddComponent<NearInteractionTouchable>();
+
+            var nearIT = cube.GetComponent<NearInteractionTouchable>();
+            var boxCollider = cube.GetComponent<BoxCollider>();
+
+            // SetTouchableCollider to a new Box Collider
+            nearIT.SetTouchableCollider(boxCollider);
+
+            Assert.AreEqual(nearIT.Bounds.x, boxCollider.size.x);
```

Comment:
Pull these asserts into a helper method that accepts a nearIt and box collider and re-use below

Reply:
Removed those asserts because a new test was written